building/quick-start.md

```diff
@@ -16,7 +16,7 @@ You can gain access to the Tableland MVP by joining our Discord. While you're th
 Tableland is currently in private beta, so to join you’ll need to do a few more steps! Having trouble getting access? One of the core Tableland developer team members will be happy to help. Just ask in #general or #support!
 {% endhint %}
 
-Next, jump into the `#🎟-creator-whitelist` channel and post your ETH public address. It should look something like this:
+Next, jump into the `#🎟-dev-whitelist` channel and post your ETH public address. It should look something like this:
```

Comment:
Should we remove this whitelisting referencing?

Reply:
yes